sdk/scheduler/src/main/java/com/mesosphere/sdk/specification/yaml/YAMLToInternalMappers.java

```diff
@@ -306,6 +307,27 @@ private static PodSpec convertPod(
         return builder.build();
     }
 
+    /**
+     * Converts a comma separated list of URIs to URI objects.
+     */
+    @VisibleForTesting
+    static Collection<URI> getUris(String uriStr) throws URISyntaxException {
+        // Separate comma separated list
+        List<String> uriStrs = Arrays.stream(uriStr.split(","))
```

Comment:
Should we extract the `","` to a constant? Probably not since it's only used here.